apps/condo/domains/notification/schema/PushToken.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createschema notification.PushToken PushToken`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/notification/access/PushToken')
+
+const { DEVICE_SERVICE_TYPES_KEYS } = require('../constants')
+
+
+const PushToken = new GQLListSchema('PushToken', {
+    schemaDoc: 'Push token',
```

Comment:
Please, explain, for what purpose this schema is created for.

Reply:
+1 Imagine how a developer looking at this description will understand this functionality.
It's out autogenerated documentation. It should be developers friendly

Reply:
fixed